src/display/api.js

```diff
@@ -314,8 +316,9 @@ function getDocument(src) {
           standardFontDataUrl &&
           isValidFetchUrl(cMapUrl, document.baseURI) &&
           isValidFetchUrl(standardFontDataUrl, document.baseURI));
+  const enableHWA = src.enableHWA === true;
```

Comment:
Please move this line up, to e.g. *just after* this line:
https://github.com/mozilla/pdf.js/blob/341ff40e74259d304665efba997484f819ef2d0d/src/display/api.js#L299

Note that the parameters in this "list" are described by the following comment, which isn't really accurate for `enableHWA`:
https://github.com/mozilla/pdf.js/blob/341ff40e74259d304665efba997484f819ef2d0d/src/display/api.js#L301